<ui version="4.0" >
 <class>RobotControl</class>
 <widget class="QWidget" name="RobotControl" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>957</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>RobotControls</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="topWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Robot Controls</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Temperatures</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_10" >
              <property name="text" >
               <string>Motor Left</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="motorTempLeft" >
              <property name="suffix" >
               <string> °C</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_11" >
              <property name="text" >
               <string>Motor Right</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="motorTempRight" >
              <property name="suffix" >
               <string> °C</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Battery</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_12" >
              <property name="text" >
               <string>Voltage</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="batteryVoltage" >
              <property name="suffix" >
               <string> V</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Speed</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_13" >
              <property name="minimumSize" >
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Left</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="speedBoxLeft" >
              <property name="suffix" >
               <string> m/s</string>
              </property>
              <property name="decimals" >
               <number>4</number>
              </property>
              <property name="maximum" >
               <double>2.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="speedBarLeft" >
              <property name="value" >
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_14" >
              <property name="minimumSize" >
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Right</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="speedBoxRight" >
              <property name="suffix" >
               <string> m/s</string>
              </property>
              <property name="decimals" >
               <number>4</number>
              </property>
              <property name="maximum" >
               <double>2.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="speedBarRight" >
              <property name="value" >
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Remote Control</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QSlider" name="remoteControlSpeed" >
              <property name="minimum" >
               <number>-36</number>
              </property>
              <property name="maximum" >
               <number>36</number>
              </property>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSlider" name="remoteControlSteering" >
            <property name="minimum" >
             <number>-36</number>
            </property>
            <property name="maximum" >
             <number>36</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Simulation</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Pose</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="poseX" >
              <property name="suffix" >
               <string> m</string>
              </property>
              <property name="minimum" >
               <double>-500.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>500.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="poseY" >
              <property name="suffix" >
               <string> m</string>
              </property>
              <property name="minimum" >
               <double>-500.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>500.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="poseAngle" >
              <property name="suffix" >
               <string>°</string>
              </property>
              <property name="maximum" >
               <double>360.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>PathLength</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelPathLength" >
              <property name="text" >
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxBrakes" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Brakes set?</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Max noise</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="noise" >
              <property name="suffix" >
               <string>%</string>
              </property>
              <property name="maximum" >
               <number>500</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="chkKeepInView" >
              <property name="text" >
               <string>keep Robot in View</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btnClearTrace" >
            <property name="text" >
             <string>Clear Trace</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Curves</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Curve coordinates:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textCurve" >
         <property name="html" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">5.20 / 10.00&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">5.19 / 9.37&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">5.19 / 8.74&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">5.58 / 8.31&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">6.16 / 8.29&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">6.73 / 8.28&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">7.30 / 8.28&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">7.87 / 8.28&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">8.44 / 8.27&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">9.16 / 8.24&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">9.88 / 8.22&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">10.59 / 8.19&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">11.31 / 8.16&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">12.03 / 8.13&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">12.75 / 8.11&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">13.47 / 8.08&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">14.19 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">14.76 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">15.63 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">16.50 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">17.37 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">18.24 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">18.63 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">19.21 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">19.89 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">20.56 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">21.04 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">21.61 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">22.39 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">23.17 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">23.59 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">24.17 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">24.88 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">25.59 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">26.03 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">26.61 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">27.37 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">28.12 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">28.51 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">29.09 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">29.88 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">30.67 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">31.01 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">31.59 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">32.19 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">32.79 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">33.21 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">33.79 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">34.61 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">35.44 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">36.26 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">37.09 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">37.43 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">38.01 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">39.00 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">39.99 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">40.98 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">41.98 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">42.31 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">42.89 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">43.66 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">44.42 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">44.76 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">45.34 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">46.29 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">47.24 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">48.20 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">49.15 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">49.49 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">50.07 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">51.06 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">52.05 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">53.05 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">54.04 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">54.38 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">54.95 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">55.29 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">55.87 / 8.05&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">56.40 / 7.79&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">56.94 / 7.53&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">57.47 / 7.26&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">58.00 / 7.00&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QCheckBox" name="chkShowCubic" >
           <property name="text" >
            <string>Show cubic spline</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QCheckBox" name="chkShowHermite" >
           <property name="text" >
            <string>Show hermite spline</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="hermiteTension" >
           <property name="minimum" >
            <double>-20.000000000000000</double>
           </property>
           <property name="maximum" >
            <double>20.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Tension</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="hermiteBias" >
           <property name="minimum" >
            <double>-10.000000000000000</double>
           </property>
           <property name="maximum" >
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Bias</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnUpdateCurves" >
         <property name="text" >
          <string>Update</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
